qa/tox.ini

```diff
@@ -1,8 +1,16 @@
 [tox]
-envlist = flake8
+envlist = flake8-py2, flake8-py3
 skipsdist = True
 
-[testenv:flake8]
+[testenv:flake8-py2]
+basepython = python2
 deps=
   flake8
 commands=flake8 --select=F,E9 --exclude=venv,.tox
+
+[testenv:flake8-py3]
+basepython = python3
+deps=
```

Comment:
nit, instead of repeating "deps" and "command", we could extract them into `[testenv]`, like
```ini
[testenv]
deps = flake8
commands = flake8 --select=F,E9 --exclude=venv,.tox
```